File: Logic Building Assignments/Logic Building Assignment 5/Assignment5_5.c
//  5. Write a program which accept N and print first 5 multiples of N.

#include<stdio.h>

void MultipleDisplay(int iNo)
{
    int i = 0;

    for(i = 1; i <= 5; i++)
    {
        printf("%d\t",(i*iNo));
    }
}

int main()
{
    int iValue = 0;

    printf("Enter a number : ");
    scanf("%d",&iValue);

    MultipleDisplay(iValue);

    return 0;
}

/*

    Output:
    D:\Marvellous Infosystems\Logic Building with Industrial Project Batch\Logic Building Assignments\Logic Building Assignment 5>gcc Assignment5_5.c

    D:\Marvellous Infosystems\Logic Building with Industrial Project Batch\Logic Building Assignments\Logic Building Assignment 5>a
    Enter a number : 4 
    4       8       12      16      20

*/

File: Python Assignments/Python Assignment 5/Assignment5_5.py
#   5. Write a recursive program which accept number from user and return
#   its factorial.

fact = 1

def factorial(No):
    global fact
    if No != 0:
        fact = fact*No
        factorial(No-1)
    return fact

def main():
    Number = int(input("Enter your number : "))
    Result = factorial(Number)
    print("Factorial of {0} is {1}".format(Number,Result))

if __name__ == "__main__":
    main()

#   Output:
#   D:\Python\Marvellous Python\Python Assignments\Python Assignment 5>Python Assignment5_5.py
#   Enter your number : 5
#   Factorial of 5 is 120